spec/classes/profile/kubernetes/accountapp_spec.rb

```diff
@@ -1,73 +1,86 @@
 require 'spec_helper'
```

Comment:
As far as I can tell, noone of the RSpec changes appropriately test the behavior of multiple `$clusters` defined. If I missed it, please point it out to me, otherwise I think some test coverage for that scenario should be added.

Reply:
You are right. I removed some tests because it was not working as expected. I still have some test to write for that